### DISCUSSION TITLE
Import not working

### DISCUSSION THREAD

**Discussion by LauraGOP** [monarc-project/MonarcAppFO#474] [Q&A]
Hello,

I performed a risk analysis on my instance. I want to export this analysis to another instance. It seems that it's not working.
![Screenshot 2022-12-14 110623](https://user-images.githubusercontent.com/101174751/207568567-8594be7b-bb33-43b8-b3a1-ae99889853b7.png) 

So I tried re-importing it on my own instance to see whether the issue was with the assessment or the other instance. And it's also not working on my instance. I added a screenshot of the message I received when trying to re-import it on my instance.

Can you please tell me what the issue is?

Kind regards,

**Comment by ruslanbaidan** [comment]
Hello,

The reason could be due to the file size. What is the size of the exported json ?

There are recommended php.ini settings to ensure the big files are accepted and import process works:
https://github.com/monarc-project/MonarcAppFO/blob/master/vagrant/bootstrap.sh#L22-L26

**Reply by LauraGOP** [reply]
It's 925 KB (947.871 bytes)

**Reply by LauraGOP** [reply]
Hello,

We tried this but the problem remains the same.

Thanks

**Reply by ruslanbaidan** [reply]
> It's 925 KB (947.871 bytes)

The file size is not big, so there is something else that prevents the process execution.

**Reply by ruslanbaidan** [reply]
Could you tell if the file is protected with password ?

**Reply by LauraGOP** [reply]
No it wasn't

**Reply by ruslanbaidan** [reply]
In this case we need the file to investigate the reason of the error.

**Reply by ruslanbaidan** [reply]
What is the Monarc version do you use for import and export ?

**Reply by jfrocha** [reply]
Check also on the password field is not autocompleted by the browser, if it is the case, delete it and import.
